src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import { Helmet } from 'react-helmet';
+import PropTypes from 'prop-types';
+import { slide as Menu } from 'react-burger-menu';
+import favicon from '../../images/favicon.ico';
+import '../../stylesheets/components/Navigation.scss';
+
+const Navigation = ({ data, children }) => {
+  // Define the meta title and description
+  const { title, description } = data.site.siteMetadata;
+
+  return (
+    <>
+      <Helmet>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0" />
+
+        <title>{ title }</title>
+        <meta name="description" content={description} />
+        <link rel="icon" href={favicon} />
+      </Helmet>
+      <div className="app">
+        <Header />
+        { children }
+        <Footer />
+      </div>
+    </>
+  );
+};
+
+Navigation.propTypes = {
+  data: PropTypes.shape({
+    site: PropTypes.shape({
+      siteMetadata: PropTypes.shape({
+        title: PropTypes.string.isRequired,
+        description: PropTypes.string.isRequired,
+      }).isRequired,
+    }).isRequired,
+  }).isRequired,
+  children: PropTypes.element.isRequired,
+};
+
+
+const Header = () => (
+  <header className="site--header">
+    <div className="site--header--wrapper">
+      <h1 className="site--header--logo"><Link to="/">ShelterTech</Link></h1>
+      <nav className="site--header--nav">
+        <SlideMenu />
+        <ul>
```

Comment:
It doesn't look like you put any `<li>` tags within the `<Links>` component, so it doesn't make sense to put a `<ul>` tag around it. `<ul>` elements are only really supposed list items within them, but if you don't actually create any, you shouldn't use it.